BitwardenShared/Core/Platform/Utilities/Constants.swift

```diff
@@ -89,6 +89,14 @@ enum Constants {
     /// The default number of KDF iterations to perform.
     static let pbkdf2Iterations = 600_000
 
+    /// The minimum number of seconds an account must have existed for before
+    /// we display the two-factor notice.
+    static let twoFactorNoticeMinimumAccountAgeInterval: TimeInterval = 86400 * 7 // Seven days
+
+    /// The minimum number of seconds after a user delays the two-factor notice
+    /// before we display the notice again.
+    static let twoFactorNoticeDelayInterval: TimeInterval = 86400 * 7 // Seven days
+
```

Comment:
♻️ Maybe for another PR, but wanted to bring this up before I forget it that it would be pretty cool to have static methods from `TimeInterval` extensions so one can specify the time unit to create the interval, e.g. `TimeInterval.days(7)` or `TimeInterval.minutes(60)` and then it calculates the interval internally which would be much easier to use.

Reply:
Agreed, though figuring out the best API surface for that is always the trick—especially as in a number of cases you actually want to go [through the Calendar object](https://developer.apple.com/documentation/foundation/nscalendar/1409577-date) to account for things like DST, and having too easy an API for "just add five days" very much can run afoul of those sorts of issues (I've had it happen before).

Reply:
Yeah makes sense. I mean if we were using iOS 16 as minimum we could use [Duration](https://developer.apple.com/documentation/Swift/Duration) object when we want high precision and avoid some of the issues you mention but well for now we need to keep using `TimeInterval`.
And totally agree that we don't need really high precision in these places but still I think refactoring to use some extension methods in another PR will be great 😄